### PR TITLE
rct cat-manifest: show Web and API urls from consumer.json

### PR DESCRIPTION
**Pull request by evgeni**
this is useful for debugging why a satellite cannot access the distributor :)

### CODE REVIEW

**Comment by evgeni** [comment]
fwiw, I could not run the tests, as `tox` fails with

```
Processing ./.tox/dist/subscription-manager-1.21.1.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-eii53rpm-build/setup.py", line 33, in <module>
        from build_ext import i18n, lint, template
    ModuleNotFoundError: No module named 'build_ext'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-eii53rpm-build/
```

so no tests adapted in the PR

**Comment by jirihnidek** [comment]
ok to test

**Comment by evgeni** [comment]
updated tests based on jenkins results

File: test/manifestdata.py [hunk 1]
```diff
@@ -48,7 +50,9 @@
         "label": "sam",
         "manifest": true
     },
-    "uuid": "ba5ac769-207e-421c-bfd2-a23c767114af"
+    "uuid": "ba5ac769-207e-421c-bfd2-a23c767114af",
+    "urlApi":"https://subscription.rhn.stage.redhat.com/subscription/consumers/"
```
Comment (jirihnidek) [inline test/manifestdata.py]: There is missing comma at the end of the line. There should be space between `"urlApi":` and "https://...`. The space is also missing at following line.

Comment (evgeni) [reply test/manifestdata.py]: ups, too much copy paste, fixed

**Comment by jirihnidek** [comment]
ok to test

**Comment by jirihnidek** [comment]
@evgeni Please, squash commits. then I will merge it to master.

**Comment by evgeni** [comment]
@jirihnidek thanks, squashed.

**Pull request merged**